src/components/Menus/EcosystemSelectMenu/EVMConnectButton.tsx

```diff
@@ -0,0 +1,45 @@
+import { isWalletInstalledAsync } from '@lifi/wallet-management';
+import { Avatar, Typography, useTheme } from '@mui/material';
+import { useMenuStore } from 'src/stores';
+import { type Connector } from 'wagmi';
+import { ConnectButton } from './EcosystemSelectMenu.style';
+import { useAccountConnect } from 'src/hooks/useAccounts';
+
+interface EvmConnectButton {
+  walletIcon?: string;
+  evm: Connector;
+}
+
+export const EVMConnectButton = ({ walletIcon, evm }: EvmConnectButton) => {
+  const theme = useTheme();
+  const connect = useAccountConnect();
+
+  const { closeAllMenus } = useMenuStore((state) => state);
+
+  const connectHandler = async () => {
+    const identityCheckPassed = await isWalletInstalledAsync(evm.id);
+    if (!identityCheckPassed) {
+      //   onNotInstalled(connector);
```

Comment:
Is this still needed? If you want to keep this in might be good to add a comment to say why